apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -42,93 +37,101 @@ const AudioPlayer: React.FC<Props> = ({ tracks }) => {
+          </span>
+          <div className="flex-grow flex items-center relative">
+            <div
+              className="absolute left-0 -top-px rounded-l-full bg-flprimary pointer-events-none h-[8px]"
+              style={{
+                width: `${(currentTime / track.duration) * 100}%`,
               }}
-              className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center md:mr-2 order-0 md:order-2 shrink-0"
-            >
-              <BackwardIcon className="w-6 h-6 text-flprimary" />
-            </button>
-            <button
-              onClick={() => {
-                setCurrentTime(currentTime + 10);
-                audio.currentTime = currentTime + 10;
+            />
+            <input
+              type="range"
+              className={cx(`flex-grow`, styles.progressBar)}
+              min={0}
+              max={track.duration}
+              value={currentTime}
+              onChange={(e) => {
+                setCurrentTime(Number(e.target.value));
+                audio && (audio.currentTime = Number(e.target.value));
               }}
-              className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center order-2 md:order-3 shrink-0"
-            >
-              <ForwardIcon className="w-6 h-6 text-flprimary" />
-            </button>
-          </div>
-          <div className="flex-grow flex items-center">
-            <span className="text-flgray-500 w-14 flex justify-start">
-              {formatDuration(currentTime)}
-            </span>
-            <div className="flex-grow flex items-center relative">
-              <div
-                className="absolute left-0 top-[-1px] rounded-l-full bg-flprimary pointer-events-none h-[8px]"
-                style={{
-                  width: `${(currentTime / audio.duration) * 100}%`,
-                }}
-              />
-              <input
-                type="range"
-                className={cx(`flex-grow`, styles.progressBar)}
-                min={0}
-                max={audio.duration}
-                value={currentTime}
-                onChange={(e) => {
-                  audio.currentTime = Number(e.target.value);
-                }}
-              />
-            </div>
-            <span className="text-flgray-500 w-14 flex justify-end">
-              {
-                audio.duration
-                  ? formatDuration(audio.duration)
-                  : `00:00` /* TODO: after the parql types are changed, this placeholder will become the duration that comes down from the api */
-              }
-            </span>
+            />
           </div>
+          <span className="text-flgray-500 w-16 flex justify-end">
+            -{formatDuration(duration(audio, track.duration) - currentTime)}
```

Comment:
i changed this to match the native app, showing a negative display of time remaining, instead of the static full duration.